### PR TITLE
feat(slider): add label align property

### PR DESCRIPTION
**Pull request by Sarin-Udompanish**
## Description
ef-slider-marker provides 'text-align' attr/prop to configure alignment of label. Valid values are :
null (default)
left
right
center

when `show-input-field` is OFF.

Label that has value = min, text-align = left.
Label that has value = max, text-align = right.
Other labels, text-align = center.
when `show-input-field` is ON.

All labels, text-align = center.

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: documents/src/pages/elements/slider.md [hunk 1]
```diff
@@ -219,6 +219,52 @@ Set `transparent` to `--progress-color` CSS variables to hide the progress.
 </ef-slider>
 ```
 
+### Customizing marker label alignment
```
Comment (wattachai-lseg) [inline documents/src/pages/elements/slider.md]:
```suggestion
### Marker label alignment
```

File: documents/src/pages/elements/slider.md [hunk 2]
```diff
@@ -219,6 +219,52 @@ Set `transparent` to `--progress-color` CSS variables to hide the progress.
...
+By default, labels corresponding to the minimum value align left, those for the maximum value align right, and others align center.
+To customize this behavior, simply set the `label-align` to the `ef-slider-marker`.
```
Comment (wattachai-lseg) [inline documents/src/pages/elements/slider.md]:
```suggestion
To customise this behavior, simply set the `label-align` to the `ef-slider-marker`.
```

File: package-lock.json
Comment (wattachai-lseg) [inline package-lock.json:1]: Another package-lock change. Something is weird. 🤔

File: packages/elemental-theme/src/custom-elements/ef-slider-marker.less [hunk 3]
```diff
@@ -11,4 +11,10 @@
     top: 15px;
     transform: translateX(calc(-50% + @slider-step-width / 2));
   }
+  &[label-align='right'] [part='label'] {
+    transform: translateX(calc(-90% + @slider-step-width / 2));
```
Comment (wattachai-lseg) [inline packages/elemental-theme/src/custom-elements/ef-slider-marker.less]:
```suggestion
    transform: translateX(calc(-100% + @slider-step-width / 2));
```

File: packages/elements/src/slider/elements/slider.ts [hunk 4]
```diff
@@ -455,6 +455,10 @@ export class Slider extends ControlElement {
 
       marker.style.setProperty('left', `${markerPosition}%`);
 
+      if (!this.isShowInputField && !marker.labelAlign) {
+        marker.labelAlign = markerPosition === 0 ? 'left' : markerPosition === 100 ? 'right' : null;
```
Comment (wattachai-lseg) [inline packages/elements/src/slider/elements/slider.ts]: let's refactor this to `switch` instead for better readability.

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]
Nice job on the documentation 👍🏼

File: packages/elements/src/slider/elements/slider.ts (hunk 4 above)
Comment (Sarin-Udompanish) [reply packages/elements/src/slider/elements/slider.ts]: For me, I think that with just two cases, the ternary operator is more concise and straightforward.

Comment (wattachai-lseg) [reply packages/elements/src/slider/elements/slider.ts]: OK, this one is rather simple.  Marking as resolved. 👌🏼

File: packages/elemental-theme/src/custom-elements/ef-slider-marker.less [hunk 5]
```diff
@@ -9,6 +9,12 @@
 
   [part='label'] {
     top: 15px;
-    transform: translateX(calc(-50% + @slider-step-width / 2));
+    transform: translateX(calc(-50% + @slider-step-width));
```
Comment (wattachai-lseg) [inline packages/elemental-theme/src/custom-elements/ef-slider-marker.less]:
```suggestion
    transform: translateX(calc(-50% + @slider-step-width / 2));
```

Comment (wattachai-lseg) [reply packages/elemental-theme/src/custom-elements/ef-slider-marker.less]: image
`transform: translateX(calc(-50% + @slider-step-width));`


image
`transform: translateX(calc(-50% +  @slider-step-width / 2));`

**Review by wattachai-lseg — COMMENTED** [review]
Let's get it centered.

**Review by wattachai-lseg — APPROVED** [review]

**Pull request merged**